OpenChange/iCalEvent+MAPIStore.m

```diff
@@ -137,6 +137,107 @@ - (void) _setupEventAlarmFromProperties: (NSDictionary *) properties
     }
 }
 
+- (int) _updateFromAttendeeMAPIProperties: (NSArray *) recipients
+                                 withRole: (NSString *) role
+                                 outParam: (BOOL *) organizerIsSet
+{
+  NSDictionary *dict;
+  NSString *attEmail;
+  iCalPerson *person;
+  iCalPersonPartStat newPartStat;
+  NSNumber *flags, *trackStatus;
+  int i, effective;
+
+  effective = 0;
+  for (i = 0; i < [recipients count]; i++)
```

Comment:
There is an implicit cast here given that `[NSArray:count]` returns an `NSUInteger` and using `int` you are losing precision. I know that are other places that are also wrong, but I want new code has the right typing.